stories/mapit.stories.js

```diff
@@ -156,4 +156,332 @@ storiesOf("HURUmap UI|MapIt/Geography", module)
       })}
       onClickGeoLayer={action("onClickGeoLayer")}
     />
+  ))
+  .add("Colour Map", () => (
+    <MapIt
+      height="100vh"
+      indexColor={object("indexColor", {
+        "very high": "#8ed3a5",
+        high: "#29a87c",
+        moderate: "#223a07",
+        low: "#7d8c6c",
+        "very low": "#5bc17d",
+      })}
+      geoIndexMapping={array("geoIndexMapping", [
```

Comment:
```suggestion
      geoIndexMapping={object("geoIndexMapping", [
```
So that we can actually see the values rather than `[object, object...`